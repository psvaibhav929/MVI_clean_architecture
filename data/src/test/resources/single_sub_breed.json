{
  "message": {
    "breed1": ["sub1"],
    "breed2": ["sub1"]
  },
  "status": "success"
}
