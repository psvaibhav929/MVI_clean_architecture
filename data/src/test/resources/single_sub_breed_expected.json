{
  "dogs": [
    {
      "dogFullName": "Breed1",
      "dogBreedName": "breed1",
      "dogSubBreedName": "sub1"
    },
    {
      "dogFullName": "Breed2",
      "dogBreedName": "breed2",
      "dogSubBreedName": "sub1"
    }
  ]
}
